scripts/test_iOS.sh

```diff
@@ -31,7 +31,7 @@ xcodebuild test \
     -project "$IOS_PROJECT_PATH" \
     -sdk iphonesimulator \
     -scheme Unity-iPhone \
-    -destination 'platform=iOS Simulator,OS=11.0.1,name=iPhone 6'
+    -destination 'platform=iOS Simulator,OS=12.0,name=iPhone 8'
```

Comment:
How do we remember to update this when a new OS comes out?

Reply:
We can make a check list somewhere whenever a new Xcode is released in the fall of each year.